Library/Formula/jq.rb

```diff
@@ -22,7 +22,7 @@ class Jq < Formula
   depends_on "bison" => :build # jq depends on bison > 2.5
 
   def install
-    system "autoreconf", "-iv"
+    system "autoreconf", "-iv" if build.devel? || build.head?
```

Comment:
Unless it exists now, we don't need it. It's easily added in future if it becomes necessary.

Great, let's tuck those autotool deps in `head do` with the `head` url inside that block then.